lib/StatusIndicator/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const StatusIndicator = props => 
+  <span>
```

Comment:
Styling for the component is handled in `gather-content-ui` not the project that consumes it.

If needs be you can override styles using SCSS variables with the `!default` keyword. For example of this look at the `Button` component style within `gather-content-ui`.